CHANGELOG.md

```diff
@@ -1,5 +1,58 @@
+ * Qute parser does not parse operator parameters with '=' correctly. See [quarkus-ls#742](https://github.com/redhat-developer/quarkus-ls/issues/742).
+ * Indent snippet new line if LSP client doesn't support `InsertTextMode#AdjustIndentation`. See [quarkus-ls#727](https://github.com/redhat-developer/quarkus-ls/issues/727).
+ * Inlay hints fails when re-opening vscode on a qute template. See [#503](https://github.com/redhat-developer/vscode-quarkus/issues/503).
+ * Ignore synthetic methods in Qute templates. See [quarkus-ls#723](https://github.com/redhat-developer/quarkus-ls/issues/723).
+ * `ClassCastException` with code action and method part. See [quarkus-ls#717](https://github.com/redhat-developer/quarkus-ls/issues/717).
+ * `JavaFileTextDocumentService` doesn't gracefully handle lack of rename support. See [quarkus-ls#700](https://github.com/redhat-developer/quarkus-ls/issues/700).
+ * Do not give code actions for stale diagnostics. See [quarkus-ls#694](https://github.com/redhat-developer/quarkus-ls/pull/694).
+ * Update snippet activation based on API changes. See [#551](https://github.com/redhat-developer/vscode-quarkus/issues/551).
+ * `AbstractQuteTemplateLinkCollector` wrongly identifies template fields. See [quarkus-ls#683](https://github.com/redhat-developer/quarkus-ls/issues/683).
+ * Completion proposes invalid `{inject:*}` items. See [quarkus-ls#590](https://github.com/redhat-developer/quarkus-ls/issues/590).
+ * Java source code not validated upon start. See [eclipse/lsp4mp#301](https://github.com/eclipse/lsp4mp/issues/301).
+ * Improve handling of `@ConfigProperties` for validation. See [eclipse/lsp4mp#304](https://github.com/eclipse/lsp4mp/issues/304)
+ * Support for the `config_ordinal` property in `microprofile-config.properties`. See [eclipse/lsp4mp#289](https://github.com/eclipse/lsp4mp/issues/289).
+ * Property evaluation should support the environment variable default value notation. See [eclipse/lsp4mp#241](https://github.com/eclipse/lsp4mp/issues/241).
+ * Display property value when hovering over a key that isn't defined in the application. See [eclipse/lsp4mp#285](https://github.com/eclipse/lsp4mp/issues/285).
+ * REST client code lens only shows up for `GET` annotations. See [eclipse/lsp4mp#94](https://github.com/eclipse/lsp4mp/issues/94)
+ * JAXRS code lens URL should always appear above method declaration. See [eclipse/lsp4mp#194](https://github.com/eclipse/lsp4mp/issues/194)
+ * Support `microprofile-health` 3.0 and later. See [eclipse/lsp4mp#314](https://github.com/eclipse/lsp4mp/issues/314)
+ * `@ConfigProperties` validation should check the annotation's fully qualified name. See [eclipse/lsp4mp#304](https://github.com/eclipse/lsp4mp/issues/304).
+ * Disable JVM logging by default to avoid language server failure. See [#548](https://github.com/redhat-developer/vscode-quarkus/issues/548).
```

Comment:
If we want to include all the bug fixes from lsp4mp, here's also `eclipse/lsp4mp#295, `eclipse/lsp4mp#311`, `eclipse/lsp4mp#315`, `eclipse/lsp4mp#325`

Reply:
I left out those since they seemed pretty specific to lsp4mp, or didn't really fix something that would have been user-visible.

Reply:
I looked through them, and you're right, they're mostly not applicable to the end user. Sorry about that. I think we should include https://github.com/eclipse/lsp4mp/issues/325 though, it has to do with the snippet value.